roadmap/implementers-guide/src/node/utility/availability-store.md

```diff
@@ -39,130 +33,138 @@ digraph {
 	st -> inc [label = "Block\nincluded"]
 	st -> prn [label = "Stored block\ntimed out"]
 	inc -> fin [label = "Block\nfinalized"]
-	fin -> prn [label = "Block keep time\n(1 day) elapsed"]
+	inc -> st [label = "Competing blocks\nfinalized"]
+	fin -> prn [label = "Block keep time\n(1 day + 1 hour) elapsed"]
 }
 ```
 
-## Lifetime of the chunk in the storage
+## Database Schema
 
-```dot process
-digraph {
-	label = "Chunk life FSM\n\n\n";
-	labelloc = "t";
-	rankdir="LR";
+We use an underlying Key-Value database where we assume we have the following operations available:
+  * `write(key, value)`
+  * `read(key) -> Option<value>`
+  * `iter_by_prefix(prefix) -> Iterator<(key, value)>` - gives all keys and values in lexicographical order starting with `prefix`
```

Comment:
```suggestion
  * `iter_by_prefix(prefix) -> Iterator<(key, value)>` - gives all keys which start with `prefix` and associated values in lexicographical order 
```

Nit: previous phrasing implied that iteration includes keys subsequent to those which start with `prefix`. Also, clarify that `prefix` applies to the keys.